website/pages/en/network/indexing.mdx

```diff
@@ -790,6 +790,26 @@ Once an Indexer has staked GRT in the protocol, the [Indexer components](/networ
 setDelegationParameters(950000, 600000, 500)
 ```
 
+### Setting delegation parameters
+
+The `setDelegationParameters()` function in the [staking contract](https://github.com/graphprotocol/contracts/blob/main/contracts/staking/Staking.sol) is essential for Indexers, allowing them to set parameters that define their interactions with Delegators, influencing their reward sharing and delegation capacity.
+
+This function enables an Indexer to define the following parameters:
+
+- `indexingRewardCut`: Percentage of indexing rewards retained for the Indexer.
+- `queryFeeCut`: Percentage of query fees retained by the Indexer.
+- `_cooldownBlocks`: Period of time (in blocks) that need to pass to update delegation parameters.
```

Comment:
I think this is being changed soon @pcarranzav?

Reply:
Yes, the Snapshot is up for the Council to vote, and we should be deploying this relatively soon